Standards/scs-XXXX-vN-Availability-Zones-Standard.md

```diff
@@ -0,0 +1,204 @@
+Configuring those backends can already include to span one storage cluster over physical machines in different fire zones.
+In combination with internal replication a configuration is possible, that already distributes replicas from volumes over different fire zones.
+When a deployment has such a configured storage backend, it already can provide safety in case of a failure of level 3.
+
+Using Availability Zones is also possible for the storage service, but configuring AZs, when having a configuration like above will not increase safety.
+Nevertheless using AZs when having different backends in different fire zones will give customers a hint to backup volumes into storages of other AZs.
+
+Additionally when the BSI recommendations are followed, there should already be redundancy in power lines, internet connection and cooling.
+An outage of one of these physical resources will not affect the storage host and its resources for more than a minimal timeframe.
+When internal replication is used, either through the IaaS or through the storage backend itself, the outage of a single PDU and such a single rack will not affect the availability of the data itself.
+All these physical factors are not requiring the usage of an Availability Zone for Storage.
+An increase of the level of failure safety will not be reached through AZs in these cases.
+
+Still it might be confusing when having deployments with compute AZs but without storage AZs.
+CSPs may need to communicate clearly up to which failure safety level their storage service can automatically have redundancy and from which level customers are responsible for the redundancy of their data.
+
+#### AZs in Network
+
+Virtualized network resources can typically be quickly and easily set up from building instructions.
+Those instructions are stored in the database of the networking service.
+
+If a physical machine, on which certain network resources are set up, is not available anymore, the resources can be rolled out on another physical machine, without being depended on the current situation of the lost resources.
+There might only be a loss of a few packets within the affected network resources.
+
+With having Compute and Storage in a good state (e.g. through having fire zones with a compute AZ each and storage being replicated over the fire zones) there would be no downsides to omitting Availability Zones for the network service.
+It might even be the opposite: Having resources running in certain Availability Zones might prevent them from being scheduled in other AZs[^3].
+As the network resources like routers are bound to an AZ, in a failure case of one AZ all resource definitions might still be there in the database, while the implementation of those resources is gone.
+Trying to rebuild them in another AZ is not possible, because the scheduler will not allow them to be implemented in another AZ, then the one thats present in their definition.
```

Comment:
```suggestion
Trying to rebuild them in another AZ is not possible, because the scheduler will not allow them to be implemented in another AZ, than the one thats present in their definition.
```